assets/js/icons/library/arrow-down-alt2.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { SVG } from 'wordpress-components';
+
+const arrowDownAlt2 = ( { className, size, ...extraProps } ) => {
+	const iconClass = [ 'dashicon', 'dashicons-arrow-down-alt2', className ]
+		.filter( Boolean )
+		.join( ' ' );
+	return (
+		<SVG
```

Comment:
Not sure if you did or not, but I tested and some very small savings could be made if running the SVG though https://jakearchibald.github.io/svgomg/ - I do this on all icon imports I introduce.

Reply:
I just copied and pasted directly from the existing dashicons... are the savings enough to warrant implementing?

Reply:
Ya the difference is super miniscule, not worth the change imo.

Reply:
168 bytes → 138 bytes for the X

Reply:
Really? it was only 1B when I used the tool, maybe I'm using it wrong.